### PR TITLE
Changes for Submissions update

### PR DESCRIPTION
**Pull request by siemensikkema**
- use .view()
- no rendering on privateContainer

See: https://github.com/nodes-vapor/submissions/pull/29

### CODE REVIEW

**Comment by siemensikkema** [comment]
@steffendsommer could you check again?

File: Sources/AdminPanel/routes.swift [hunk 1]
```diff
@@ -9,27 +9,33 @@ public struct AdminPanelEndpoints {
...
+        adminPanelUserBasePath: String = "/admin/users",
+        dashboard: String = "/admin",
         login: String = "/admin/login",
         logout: String = "/admin/logout",
-        dashboard: String = "/admin",
-        renderAdminPanelUserList: String = "/admin/users",
-        renderCreateAdminPanelUser: String = "/admin/users/create",
-        createAdminPanelUser: String = "/admin/users/create",
-        renderEditMe: String = "/admin/users/me/edit"
+
```
Comment (steffendsommer) [inline Sources/AdminPanel/routes.swift]: Blank line

**Comment by rasmusebbesen** [comment]
Should this be merged or cancelled?

**Comment by steffendsommer** [comment]
I believe we want this in, but it probably need to be updated with the latest change. @siemensikkema

**Comment by siemensikkema** [comment]
@rasmusebbesen @steffendsommer This is outdated. I will close it.

**Pull request closed without merging**